js/scripts.js

```diff
@@ -0,0 +1,14 @@
+// If excercises
+var a = 2,
+	b = 5,
+	value = (a*a) + (2 * a * b) - (b*b);
+
+console.log(value);
+
+if (value == 0) {
```

Comment:
Nie poprawiaj, ale na przyszłość zalecam porównywać rzeczy poprzez `===` zamiast `==`. Sprawdzasz wtedy nie tylko po wartości ale także po type. Js jest głupi i jak sprawdzasz po wartości to string `'2'` jest dla niego tym samym co number `2` - co ma potencjał na wprowadzenie błędów w dalszej logice kodu.